knowledge-base/resolve-graphics-issues.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Resolving Graphics Incompatibility Issues
+description: "Test and resolve video driver incompatibility issue that prevent the proper startup of Fiddler Everywhere."
+type: how-to
+slug: troubleshoot-video-incompatibility
+publish: true
+res_type: kb
+---
+
+
+## Environment
+
+|   |   |
+|---|---|
+| Product   |
+| Product Version | Fiddler Everywhere 5.1.0 and above  |
+
+
+## Description
+
+I've installed the latest version of Fiddler Everywhere, but it fails to start, or it starts with a broken UI like a hanging splash screen, an empty white screen, etc. How can I troubleshoot and resolve the issue?
+
+## Solution
+
+One probable reason for the above issues to manifest is incompatibilities between the installed graphics drivers and the Electron framework (which is used to create the UI of the Fiddler Everywhere). To resolve the issue, you can try to update your graphics driver. The issue was previously reported to appear with some NVIDIA video cards (like Quadro) alongside specific driver versions.
+
+Alternatively, if the update of the driver does not solve the issue, you can explicitly turn off the hardware acceleration through a boolean flag called `disableHardwareAcceleration` in the `electron-settings.json` file. Fiddler will use a software output device for rendering in the CPU when the hardware acceleration is explicitly disabled.
+
+- Open the following folder 
+  ```bash
+  %userprofile%\.fiddler\Settings\electron-settings.json
```

Comment:
it's a windows specific path, on macos/linux would be ~/.fiddler/.fiddler/Settings/electron-settings.json